DeviceCode/Drivers/LargeBuffer/Test/LargeBuffer_test.cpp

```diff
@@ -15,7 +15,7 @@ void LargeBuffer_PostEvent( UINT16 marshalId )
 
 void LargeBufferTest_Completion(void* arg)
 {
-    LargeBuffer_PostEvent( (UINT16)arg );
+    LargeBuffer_PostEvent( (UINT16)(UINT32)arg );

```

Comment:
Should leave a comment why we're doing this.  I assume this gets the compiler to emit different code, but it's very unusual to cast twice.